internal/monitor/bucket.go

```diff
@@ -20,10 +20,39 @@ import (
 	"io"
 	"time"
 
+	storagev2 "cloud.google.com/go/storage"
 	"github.com/googlecloudplatform/gcsfuse/v2/common"
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/storage/gcs"
+	"go.opencensus.io/stats"
 )
 
+var (
+	// OpenCensus measures
+	readBytesCountOC = stats.Int64("gcs/read_bytes_count", "The number of bytes read from GCS objects.", stats.UnitBytes)
```

Comment:
This change doesnt look correct to me. Can you check once.

Reply:
+1 - Metrics declarations have been refactored: https://github.com/GoogleCloudPlatform/gcsfuse/blob/6f2e04bb151d740d36a09aa8134cd5d2b80a9f6e/common/oc_metrics.go#L156